### PR TITLE
fix the empty schema in storagecahce

### PR DESCRIPTION
**Pull request by RodgeFu**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2019-08-01-preview/storagecache.json [hunk 1]
```diff
@@ -526,11 +523,9 @@
         "responses": {
           "200": {
             "description": "All cache data is clean.",
-            "schema": {}
```
Comment (markcowl) [inline specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2019-08-01-preview/storagecache.json]: isn't there a schema for this POST response?

File: specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2019-08-01-preview/storagecache.json [hunk 2]
```diff
@@ -583,11 +578,9 @@
         "responses": {
           "200": {
             "description": "Cache is Active.",
-            "schema": {}
```
Comment (markcowl) [inline specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2019-08-01-preview/storagecache.json]: No schema?

File: specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2019-08-01-preview/storagecache.json [hunk 3]
```diff
@@ -1032,11 +1020,9 @@
         "responses": {
           "201": {
             "description": "Cache firmware is up to date.",
-            "schema": {}
```
Comment (markcowl) [inline specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2019-08-01-preview/storagecache.json]: no schema?

File: specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2019-11-01/storagecache.json [hunk 4]
```diff
@@ -973,15 +958,12 @@
         "responses": {
           "201": {
             "description": "Cache firmware is up to date.",
-            "schema": {}
```
Comment (markcowl) [inline specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2019-11-01/storagecache.json]: No schema?

**Review by markcowl — CHANGES_REQUESTED** [review]

File: specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2019-08-01-preview/storagecache.json (hunk 1 above)
Comment (RodgeFu) [reply specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2019-08-01-preview/storagecache.json]: I think it's not needed if no response data expected. empty schema here is not valid and will be complained by autorest. Do you know how to include service team to validate the change? thanks.

**Comment by JackTn** [comment]
Hi, @RodgeFu. The PR has be closed for a long time and it's related branch still exist. If no further used for over 14 days, I will delete this related branch. Please tell me if you still need this branch.

**Comment by JackTn** [comment]
Hi, @RodgeFu. The PR has be closed for a long time and it's related branch still exist. Please tell me if you still need this branch or i will delete it in 14 days.

**Pull request closed without merging**